### DISCUSSION TITLE
H-Key support

### DISCUSSION THREAD

**Discussion by freezeware** [shanteacontrols/OpenDeck#161] [Ideas]
I was unable to find any information regarding to connecting a keyboard to it with notes / velocity data etc. Albeit I didn't spend a lot of time looking for it, so apologizes if that exists. Is, or will this be supported?

**Comment by paradajz** [comment]
I'm not familiar with H-key. Can you provide more info?

**Comment by freezeware** [comment]
I'm sorry for the confusion, I mean the normal white keys on a regular MIDI keyboard.

**Reply by paradajz** [reply]
I see. No, that's not supported.

**Reply by freezeware** [reply]
Would be a great thing to have...
